heartbeat/podman-etcd

```diff
@@ -347,21 +347,41 @@ remove_container()
 	return $rc
 }
 
+# Correctly wraps an ipv6 in [] for url otherwise use return normal ipv4 address.
+ip_url() {
+	local ip_addr=$1
+	local value
+	if [ $ip_addr == *":"* ]; then
+		value="[$ip_addr]"
+	else
+		value="$ip_addr"
+	fi
+	echo "https://$value"
+}
+
 attribute_node_ip()
 {
 	local action="$1"
 	local attribute="node_ip"
-	local value
+	local value ip_addr name
```

Comment:
`value` can be removed, as it's no longer used in this function.

Reply:
Ah, good catch, removed